Tests/swift-sdk-swift-tests/IterableInboxNavigationViewControllerTests.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  IterableInboxNavigationViewControllerTests.swift
+//  swift-sdk-swift-tests
+//
+//  Created by Jay Kim on 6/4/19.
+//  Copyright © 2019 Iterable. All rights reserved.
+//
+
+import XCTest
+
+@testable import IterableSDK
+
+class IterableInboxNavigationViewControllerTests: XCTestCase {
+    func testInitWithNoRootViewController() {
+        let inboxNavigationVC = IterableInboxNavigationViewController()
+        
+        XCTAssertNotNil(inboxNavigationVC.viewControllers[0] as? IterableInboxViewController)
+    }
+    
+    func testInitWithCoder() {
+        guard let inboxNavigationVC = IterableInboxNavigationViewController(coder: NSKeyedUnarchiver(forReadingWith: Data())) else {
+            XCTFail()
+            return
+        }
+        
+        XCTAssertNil(inboxNavigationVC.navigationController?.tabBarItem.badgeValue)
```

Comment:
This will always be nil, since `inboxNavigationVC` has no parent navigation controller.